src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -72,7 +72,9 @@ public BasePageObject(WebDriver driver) {
 	}
 
 	protected void setWindowSize() {
```

Comment:
either parameterized this method or change the method name to be specific to android